scrapers/sd/events.py

```diff
@@ -179,12 +182,16 @@ def scrape(self):
             for key in events_by_date:
                 yield events_by_date[key]
 
-    def create_event(self, committee, agenda_document):
+    def create_event(self, committee, agenda_document, location=""):
         name = committee["FullName"]
 
         start_date = dateutil.parser.parse(agenda_document["DocumentDate"])
 
-        location = "500 E Capitol Ave, Pierre, SD 57501"
+        base_loc = "500 E Capitol Ave, Pierre, SD 57501"
+        if location and re.match(r"Room \d+", location):
+            location = base_loc + ", " + location
```

Comment:
Think this should be `location = location + ", " + base_loc` just for readability

Reply:
@NewAgeAirbender I've fixed this issue.